### DISCUSSION TITLE
Recording sample into kit row resets kit row settings

### DISCUSSION THREAD

**Discussion by seangoodvibes** [SynthstromAudible/DelugeFirmware#2998] [Ideas]
### Please describe the problem:

Recording a sample into a kit row using audition pad + record will reset the kit row's settings to the default sample drum settings.

This behaves differently from loading a sample into a kit row using shift / audition + browse which preserves the kit row's settings.

### What is the expected behavior?

Provide an alternate option to preserve kit row settings when recording a sample into a kit row.

### What hardware did you reproduce it with?

OLED

### What firmware did you reproduce it with?

Nightly

**Comment by m-m-adams** [comment]
I think this is a feature request. The current behaviour is better for the normal case of making a row and recording into it since creating a row will grab any existing drum from memory

**Comment by duncanrmi** [comment]
imagine if your row settings, that you don't want to change, made some drastic difference to the sound, like a big pitch shift or a reversal or something- that would potentially be confusing if you sampled something new & then heard it back transformed.
no?

**Reply by seangoodvibes** [reply]
> imagine if your row settings, that you don't want to change, made some drastic difference to the sound, like a big pitch shift or a reversal or something- that would potentially be confusing if you sampled something new & then heard it back transformed. no?

This is not meant as a permanent change in behaviour, but an optional behaviour setting.

If you're recording takes into a kit row live it can be really annoying to have the volume of that take constantly change. It makes mixing in live samples difficult.

You can for example put notes in a kit note row and resample live to replace samples live and have them immediately play back. In this use case, you don't want the replaced sample to have a drastic change in volume.

**Reply by duncanrmi** [reply]
thanks, sean... gotcha.
if I have a complaint about the deluge at all, it's that it can & does support so many different workflows that one can completely overlook a particular use-case. my excuse is that I'm a bit old-school & tend towards the paradigm of 'sampling' being part of a preparatory phase, rather than during a performance.
optional is good.

(last week, I snagged a prophet 2000 sampling keyboard, finally replacing the one I had in the 90s, upon which I spent thousands of hours. the muscle-memory came right back, but where to find blank floppies in 2024? :-)  some of those original samples are now in the deluge as multisample sets... )

**Reply by ckbarlow** [reply]
This – persistent kit row settings – is exactly what I came here to request. I used to live-sample into an ASR-10, where I'd set up a great template to sample into – stuff like mod wheel = loop start, aftertouch = filtering, etc. I would **love** if I could use the Deluge similarly; I would incorporate it into my current performance rig.
